release-notes/6.0/preview/api-diff/preview7/.Net/6.0.0-preview7_System.ComponentModel.Design.md

```diff
@@ -0,0 +1,17 @@
+# System.ComponentModel.Design
+
+``` diff
+ namespace System.ComponentModel.Design {
+     public class DesignerVerbCollection : CollectionBase {
+-        protected override void OnClear();
+
+-        protected override void OnInsert(int index, object value);
+
+-        protected override void OnRemove(int index, object value);
+
+-        protected override void OnSet(int index, object oldValue, object newValue);
```

Comment:
These are a little strange.  I see they were removed from src in https://github.com/dotnet/corefx/pull/30159 several years ago, and then they were just removed from the ref in https://github.com/dotnet/runtime/commit/885296a7da5f2699195df37e97e261f46d20faf2#diff-6466df6aea2f15578eae754246e298f9fd3739cc4e81eb7736977ba7efde18e0L1723-L1726. The removal doesn't actually change behavior because the overrides had been empty and were just overriding empty virtuals on the base.

I'd suggest we just remove this whole file from the diff, but @dotnet/area-system-componentmodel can best comment on it.

Reply:
cc @dotnet/area-system-componentmodel